ministryofjustice/laa-review-criminal-legal-aid
# Do not display premium bonds if not answered in apply

## Description of change
It is now possible for certain applications to not go through the full capital means assessment, meaning the premium bonds question may not be asked and can therefore be null whereas it was previously always asked

This pr amends the view to not display the premium bonds section if the question was not asked in apply

## Link to relevant ticket

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:

**Broken when premium bonds is not submitted with capital details** 

<img width="756" alt="Screenshot 2024-04-03 at 12 49 26" src="https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/51047911/dad25d42-fa5c-43be-86dd-9cee6b335cf4">

### After changes:
**With no premium bonds details** 

<img width="887" alt="Screenshot 2024-04-03 at 12 46 41" src="https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/51047911/a742a7cc-8e1b-4678-9dbd-835dd3ce6e85">

**With premium bonds details** 

<img width="830" alt="Screenshot 2024-04-03 at 09 21 29" src="https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/51047911/faa29027-11b8-435a-9e7e-39c3edf5845e">

<img width="810" alt="Screenshot 2024-04-03 at 09 26 15" src="https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/51047911/60ddf848-c483-441a-966b-e9aeafa1340b">

## How to manually test the feature
Start an application in apply that has a summary only case type 
When you are going through the means screens you should not be asked the premium bonds question 
Submit the application and confirm that no premium bonds section is shown in review 

Also submit an application (e.g. either way case type) with premium bonds information and confirm the premium bonds is still displaying as expected
